sample.redhat-ci.yml

```diff
@@ -83,6 +83,28 @@ env:
     VAR1: val1
     VAR2: val2
 
+# OPTIONAL
+# If the project follows the Build API (see
+# https://github.com/cgwalters/build-api), you may specify
+# the 'build' key to automatically build the code.
+build: true
+
+# You may customize the build further by providing a dict
+# instead.
+build:
+    # OPTIONAL
+    # Customize the configuration step.
+    config-opts: >
+        --prefix=/usr
+        --libdir=/usr/lib64
+        CFLAGS='-ggdb -O0'
+    # OPTIONAL
+    # Customize the build step.
+    build-opts: -j4
```

Comment:
I think parallelization is something that should be injected by the host system, not chosen per user.  We should just default to `-j $(getconf _NPROCESSORS_ONLN)`.